### ISSUE TITLE
Postico does not save height of favourites window after restart

### ISSUE THREAD

**Issue by m0n9oose** [jakob/Postico#602]
You can use this template for reporting bugs.
Feel free to remove parts that are not relevant, or just write something free-form if you prefer.

## What did you do?

- Launch Postico app
- Increase Favourites window height to full height of screen
- Quit the Postico
- Launch Postico

## What did you expect to happen?

- Expected to see exactly the same Favourites window size

## What actually happened?

- Height of window is slightly decreased. And it's getting smaller on each relaunch.

## What software versions are you using?

Postico version:  
Version 1.4.2 (2845)

macOS version:  
10.14 (18A391)

PostgreSQL version:  
PostgreSQL 10.5 on x86_64-apple-darwin17.7.0, compiled by Apple LLVM version 9.1.0 (clang-902.0.39.2), 64-bit

Hardware:
MacBook Pro (13-inch, 2017)

**Comment by jakob** [comment]
Can you try this build?

[postico-3717.zip](https://github.com/jakob/Postico/files/2533884/postico-3717.zip)

**Comment by m0n9oose** [comment]
@jakob tested this build, everything's fine. Thanks a lot!

**Comment by fxer** [comment]
I take it this is separate from restoring the size of the split-pane inside the SQL query view? The slider always defaults back to up near the top of the window (~15% query pane, ~85% result pane)

**Comment by jakob** [comment]
@fxer Yes, that is a separate issue. I thought someone already complained about that, but I can't find the issue any more. Can you open a new issue?
